components/molecules/ContributorProfileHeader/contributor-profile-header.tsx

```diff
@@ -43,15 +67,25 @@ const ContributorProfileHeader = ({ avatarUrl, githubName, isConnected }: Contri
           />
         </div>
         {isConnected && (
-          <div className="flex md:translate-y-0 translate-y-28 gap-3 flex-col md:flex-row items-center">
-            <a target="_blank" rel="noopener noreferrer" href={`https://github.com/${githubName}`}>
-              <MarkGithubIcon size={24} className="visible xs:invisible mr-2 mb-3" />
-              <Button className="invisible xs:visible px-5 py-2 mb-6 bg-white" variant="text">
-                View on GitHub
+          <div className="flex flex-col items-center gap-3 md:translate-y-0 translate-y-28 md:flex-row">
+            {user ? (
+              username !== user?.user_metadata.user_name && (
```

Comment:
This should be a prop that is passed is, such as `isOwner` and done from outside this component.

```suggestion
              username !== user?.user_metadata.user_name && (
```